templates/page.html

```diff
@@ -57,10 +57,10 @@
                   <a href="https://exist-db.org">eXist-db.org</a>
                 </li>
                 <li>
-                  <a href="documentation.xml">Documentation</a>
+                  <a href="https://exist-db.org/exist/apps/doc/documentation.xml">Documentation</a>
```

Comment:
Oh, so that's where the link should be going :)

Reply:
yup